pulp_rpm/tests/functional/constants.py

```diff
@@ -35,15 +35,35 @@
 This matches the format output by the "content_summary" field on "../repositories/../versions/../".
 """
 
-RPM_SIGNED_URL = urljoin(RPM_SIGNED_FIXTURE_URL, 'bear-4.1-1.noarch.rpm')
-"""The path to a single signed RPM package."""
-
-RPM_UNSIGNED_URL = urljoin(RPM_UNSIGNED_FIXTURE_URL, 'bear-4.1-1.noarch.rpm')
-"""The path to a single unsigned RPM package."""
 
 RPM_PACKAGE_NAME = 'bear'
 """The name of one RPM package."""
 
+RPM_PACKAGE_FILENAME = 'bear-4.1-1.noarch.rpm'
+"""The filename of one RPM package."""
+
+RPM_PACKAGE_DATA = {
+    'name': 'bear',
+    'epoch': '0',
+    'version': '4.1',
+    'release': '1',
+    'arch': 'noarch',
+    'release': '1',
```

Comment:
Duplicate `key` in dictionary.

Reply:
@kersommoura thanks, I'll fix it before merging